string object_id
---
bool success
string error_msg